test/e2e/autoscale.go

```diff
@@ -288,25 +288,25 @@ func assertScaleDown(ctx *TestContext) {
 	ctx.logf("Scaled down.")
 }
 
-func numberOfReadyPods(ctx *TestContext) (float64, error) {
+func numberOfReadyPods(ctx *TestContext) (float64, *corev1.Endpoints, error) {
 	// SKS name matches that of revision.
 	n := ctx.resources.Revision.Name
 	sks, err := ctx.clients.NetworkingClient.ServerlessServices.Get(context.Background(), n, metav1.GetOptions{})
 	if err != nil {
 		ctx.logf("Error getting SKS %q: %v", n, err)
-		return 0, fmt.Errorf("error retrieving sks %q: %w", n, err)
+		return 0, nil, fmt.Errorf("error retrieving sks %q: %w", n, err)
 	}
 	if sks.Status.PrivateServiceName == "" {
 		ctx.logf("SKS %s has not yet reconciled", n)
 		// Not an error, but no pods either.
-		return 0, nil
+		return 0, &corev1.Endpoints{}, nil
```

Comment:
`nil` here too? The printer below seems to deal with this?

Reply:
I was attempting to distinguish between "API call returned error" and "have a thing, but it's empty" cases. Let me fill in a bit more here to clarify.